libr/anal/p/anal_x86_cs.c

```diff
@@ -67,6 +67,37 @@ static bool is_xmm_reg(cs_x86_op op) {
 	}
 }
 
+char *reg32to64 (char *reg32)
+{
+	ut16 reg_s;
```

Comment:
This shoudl be done using the rreg api. But this is clearly faster and x86 specific